packages/react-components/react-skeleton/src/components/Skeleton/Skeleton.types.ts

```diff
@@ -1,17 +1,43 @@
 import type { ComponentProps, ComponentState, Slot } from '@fluentui/react-utilities';
 
 export type SkeletonSlots = {
-  root: Slot<'div'>;
+  /**
+   * The root slot of the `Skeleton` is the container that will contain the slots that make up a `Skeleton`
+   * and any data that the `Skeleton` will load. The default html element is a `div`.
+   */
+  root: NonNullable<Slot<'div'>>;
 };
 
 /**
  * Skeleton Props
  */
-export type SkeletonProps = ComponentProps<SkeletonSlots> & {};
+export type SkeletonProps = Omit<ComponentProps<Partial<SkeletonSlots>>, 'width'> & {
+  /**
+   * Sets the width value of the skeleton wrapper.
+   * @defaultValue 100%
+   */
+  width?: number | string;
+
+  /**
+   * Sets the appearance of the Skeleton.
+   * @defaultValue opaque
+   */
+  appearance?: 'opaque' | 'translucent';
+
+  /**
+   * The animation type for the Skeleton
+   * @defaultValue wave
+   */
+  animation?: 'wave' | 'pulse';
```

Comment:
_nit:_ Alphabetically order props

```suggestion
  /**
   * The animation type for the Skeleton
   * @defaultValue wave
   */
  animation?: 'wave' | 'pulse';
  
  /**
   * Sets the appearance of the Skeleton.
   * @defaultValue opaque
   */
  appearance?: 'opaque' | 'translucent';
  
  /**
   * Sets the width value of the skeleton wrapper.
   * @defaultValue 100%
   */
  width?: number | string;
```